tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/AbstractPortalBundle.java

```diff
@@ -184,6 +188,64 @@ protected String getHttpPortValue(
         return null;
     }
 
+    protected String setHttpPortValue(
```

Comment:
This only works for Tomcat, so it shouldn't be in the AbstractPortalBundle class but rather the PortalTomcatBundle class.